master.js

```diff
@@ -334,7 +334,13 @@ class AlAzureMaster {
                 });
             }),
             async.reflect(function(callback) {
-                return master._alAzureDlBlob.getDlBlobStats(callback);
+                return master._alAzureDlBlob.getDlBlobStats(function(err, dlstats) {
+                    if (err && err.statusCode === 404) {
```

Comment:
Is this because the customer removed the dlblob container? If so, should we maybe switch the collector's status to warning with a message to the effect?

Reply:
I haven't pushed the latest changes.  That was the intent.